kbase-extension/static/kbase/js/widgets/narrative_core/upload/stagingAreaViewer.js

```diff
@@ -264,13 +308,17 @@ define([
             files = files || [];
             const emptyMsg = 'No files found.';
 
+            // stagingAreaViewer.identifyImporterMappings(files).then(function (filesWithMappings) {
```

Comment:
This can be deleted, I think, as you have it handled above.

Reply:
done